### PR TITLE
feat : Add typography component

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
added header tag sizing , paragraph tag sizing, colored text , center align text in  typography.css

### CODE REVIEW

**Review by stutisk — COMMENTED** [review]
looks good to me !

File: Components/typography.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.text-primary{color: var(--secondary-hover)}
+
+.text-secondary{color: var(--secondary-color)}
+
+.text-st{text-align:left}
+
+.text-md{text-align: center}
+
+.text-ed{text-align: right}
```
Comment (JantuDeb) [inline Components/typography.css]: Name of these classes are not self-explanatory, it is unclear whether these classes are for size or alignment.
You can use `text-center`, `text-start`, `text-end`

**Review by JantuDeb — APPROVED** [review]
Suggested some changes

File: Components/typography.css (hunk 1 above)
Comment (Omkar-Chaskar) [reply Components/typography.css]: done. thanks for correcting me.

**Pull request merged**